src/index.ts

```diff
@@ -20,17 +20,28 @@ function retryMyFetch(http: Fetch, params: decoratorOptions): Fetch {
     try {
       const { timeout = 1000 } = params;
       const defaultRefreshCallback: beforeRefetchInterface = () => sleep(timeout);
-      const { beforeRefetch = defaultRefreshCallback, maxTryCount = 5 } = params;
+      const {
+        beforeRefetch = defaultRefreshCallback,
+        maxTryCount = 5,
+        doNotRefetchIfStatuses,
```

Comment:
I think, `doNotRefetchIfStatuses` does not disclose its intention. Does `doNotRetryIfStatuses` sound better? 
It means that we do not call `retry` method in case when the status code fits our condition.